src/api/student.js

```diff
@@ -29,6 +29,10 @@ app.post('/', (req, res) => {
                 email: s['E-mail'],
                 major: s['Major']
             };
+            if (student.name === undefined || student.name === undefined || student.status === undefined ||
+                    student.email === undefined || student.major === undefined) {
+                return res.failureJson('File not in the correct format!');
```

Comment:
NACK: please set the [status code](http://expressjs.com/en/4x/api.html#res.status) to something descriptive, like 400 Bad Request.